packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -91,7 +85,6 @@ contract SuperchainConfig is Initializable, ISemver {
     /// @notice Constructs the SuperchainConfig contract.
```

Comment:
nit: semver bump

Reply:
DelayedVetoable needs a bump, yes, but this contract isn't on develop yet, so I think it should be OK at 1.0.0, right?

Reply:
bumped it anyways